### PR TITLE
docs(updated v1.2.0 cli subcommands): updated the subcommands for the v1.2.0 cli page

### PR DESCRIPTION
**Pull request by crisgarner**
# Description

updated the subcommands for the v1.2.0 cli page

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: website/versioned_docs/version-v1.2/cli.md [hunk 1]
```diff
@@ -27,28 +27,28 @@ pnpm run hardhat
...
...PoseidonT6 contract address <br/> `-g, --signupGatekeeperAddress <signupGatekeeperAddress>`: The signup gatekeeper contract address <br/> `-q, --quiet`: Whether to print values to the console <br/> `-r, --rpc-provider <provider>`: The rpc provider URL <br/> `-s, --stateTreeDepth <stateTreeDepth>`: The state tree depth <br/> `-uq, --use-quadratic-voting`: Whether to use quadratic voting                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (kittybest) [inline website/versioned_docs/version-v1.2/cli.md:32]: there are two 
`-g, --signupGatekeeperAddress <signupGatekeeperAddress>`: The signup gatekeeper contract address <br/>

should remove one of them

File: website/versioned_docs/version-v1.2/cli.md [hunk 2]
```diff
@@ -27,28 +27,28 @@ pnpm run hardhat
...
...message batch depth <br/> `-p, --process-messages-zkey <processMessagesZkeyPath>`: The process messages zkey path (see different options to use specific circuits [Trusted setup](https://maci.pse.dev/docs/trusted-setup) or [Testing](https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing)) <br/> `-t, --tally-votes-zkey <tallyVotesZkeyPath>`: The tally votes zkey path (see different options to use specific circuits [Trusted setup](https://maci.pse.dev/docs/trusted-setup) or [Testing](https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing))                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```
Comment (kittybest) [inline website/versioned_docs/version-v1.2/cli.md:33]: there's no `-uq` flag in v1.2?

there's another flag for this command:
`-ss, --subsidy-zkey <subsidyZkeyPath>`
    "the subsidy zkey path (see different options for zkey files to use specific circuits https://maci.pse.dev/docs/trusted-setup, https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing)"

we removed subsidy in current version, but there's still subsidy in v1.2

File: website/versioned_docs/version-v1.2/cli.md [hunk 3]
```diff
@@ -27,28 +27,28 @@ pnpm run hardhat
...
...dress>`: The MACI contract address <br/> `-q, --quiet`: Whether to print values to the console <br/> `-r, --rpc-provider <provider>`: The rpc provider URL                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (kittybest) [inline website/versioned_docs/version-v1.2/cli.md:39]: there's no `-uq` flag in v1.2?

missing this 
`-se, --subsidy-enabled <subsidyEnabled>` whether to deploy subsidy contract

File: website/versioned_docs/version-v1.2/cli.md [hunk 4]
```diff
@@ -27,28 +27,28 @@ pnpm run hardhat
...
...docs/testing/#pre-compiled-artifacts-for-testing)) <br/> `-tqv, --tally-votes-zkey-qv <tallyVotesZkeyPathQv>`: The tally votes qv zkey path (see different options to use specific circuits [Trusted setup](https://maci.pse.dev/docs/trusted-setup) or [Testing](https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing)) <br/> `-pnqv, --process-messages-zkey-non-qv <processMessagesZkeyPathNonQv>`: The process messages non-qv zkey path (see different options to use specific circuits [Trusted setup](https://maci.pse.dev/docs/trusted-setup) or [Testing](https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing)) <br/> `-tnqv, --tally-votes-zkey-non-qv <tallyVotesZkeyPathNonQv>`: The tally votes qv zkey path (see different options to use specific circuits [Trusted setup](https://maci.pse.dev/docs/trusted-setup) or [Testing](https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing)) <br/> `-uq, --use-quadratic-voting"`: Whether to use quadratic voting <br/> `-k, --vk-registry <vkRegistry>`: The vk registry contract address <br/> `-q, --quiet`: Whether to print values to the console <br/> `-r, --rpc-provider <provider>`: The rpc provider URL              |
```
Comment (kittybest) [inline website/versioned_docs/version-v1.2/cli.md:40]: `-pqv`, `-tqv`, `-pnqv`, `-tnqv` flags might be in the current version only? 

missing following command:
`-ss, --subsidy-zkey <subsidyZkeyPath>`
    the subsidy zkey path (see different options for zkey files to use specific circuits https://maci.pse.dev/docs/trusted-setup, https://maci.pse.dev/docs/testing/#pre-compiled-artifacts-for-testing)

**Review by kittybest — COMMENTED** [review]
Left some comments, but overall I think you actually wanna modify the v1.3.x cli page? (instead of the v1.2 cli page)

File: website/versioned_docs/version-v1.2/cli.md (hunk 4 above)
Comment (crisgarner) [reply website/versioned_docs/version-v1.2/cli.md:40]: Same here, the verifying keys on v 1.2.0 has these params 
image

**Comment by crisgarner** [comment]
> Left some comments, but overall I think you actually wanna modify the v1.3.x cli page? (instead of the v1.2 cli page)

Wanted to start with v1.2.0 as in theory should be more stable

**Comment by crisgarner** [comment]
Found the issue, most likely the build I had installed when running the help command was v1.3.0 will start again from scratch

**Pull request closed without merging**